src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java

```diff
@@ -26,49 +26,83 @@
 import com.puppycrawl.tools.checkstyle.utils.JavadocUtil;
 
 /**
- * Checks that:
+ * <p>
+ * Checks the Javadoc paragraph. Checks that:
+ * </p>
  * <ul>
  * <li>There is one blank line between each of two paragraphs
  * and one blank line before the at-clauses block if it is present.</li>
  * <li>Each paragraph but the first has &lt;p&gt; immediately
  * before the first word, with no space after.</li>
  * </ul>
- *
- * <p>The check can be specified by option allowNewlineParagraph,
- * which says whether the &lt;p&gt; tag should be placed immediately before
- * the first word.
- *
- * <p>Default configuration:
+ * <ul>
+ * <li>
+ * Property {@code violateExecutionOnNonTightHtml} - Control when to print violations
+ * if the Javadoc being examined by this check violates the tight html rules defined at
+ * <a href="https://checkstyle.org/writingjavadocchecks.html#Tight-HTML_rules">
+ * Tight-HTML Rules</a>.
+ * Default value is {@code false}.
+ * </li>
+ * <li>
+ * Property {@code allowNewlineParagraph} - Control whether the &lt;p&gt; tag
+ * should be placed immediately before the first word.
+ * Default value is {@code true}.
+ * </li>
+ * </ul>
+ * <p>
+ * To configure the default check:
  * </p>
  * <pre>
  * &lt;module name=&quot;JavadocParagraph&quot;/&gt;
  * </pre>
- *
- * <p>To allow newlines and spaces immediately after the &lt;p&gt; tag:
+ * <p>
+ * By default, the check will report a violation if there is a new line
```

Comment:
The `allowNewlineParagraph` property works the opposite way, so we need more samples here.